### ISSUE TITLE
Win 10 syncing 

### ISSUE THREAD

**Issue by cnorthcote** [stratisproject/Breeze#144]
Sync has been running for 10 hours and at 77% 6 connections.

**Comment by bokobza** [comment]
Hi, did you create or recover a wallet? If you recovered, from what date was it?

**Comment by cnorthcote** [comment]
I created the wallet. Fresh install at about 11am GMT 1/7/17. So 12 hours now

**Comment by cnorthcote** [comment]
I received two transactions. Both unconfirmed

**Comment by bokobza** [comment]
that's strange. when you create a wallet there is no need for syncing at all. 
Did you by any chance have another wallet in the stratisnode/bitcoin/testnet folder? You would if you had a previous version of the wallet. 
To give you a bit of context: what the app is that it looks for "*.wallet.json" files in that folders and loads them up as wallets. It then tries to figure out from where to sync them. If you had an older wallet there, the app is probably trying to sync this one.

**Comment by cnorthcote** [comment]
The one thing that happened after the install was windows update to 1703. I
know that screw around with programs. Created an entire image for my work.
It then did the upgrade and ruined a days worth of work. I'm not at the
laptop now but let me check for Windows.old file when I'm back. Could be it
as this was a fresh vanilla install of win 10

**Comment by cnorthcote** [comment]
No windows.old file so can't be the issue. It is now 100% 8 connections all
transactions confirmed

**Comment by bokobza** [comment]
glad all seems to be sorted.
I'm still wondering what the original issue was.
when you open the app, do you have only 1 wallet in the dropdown?

**Comment by cnorthcote** [comment]
Yes only one. Want me to create another?

**Comment by bokobza** [comment]
If you had 2 that would give me a clue to what the problem was. Now I have no idea. No need to add another one, but thank you :-)
I'll try to reproduce this bug but for now I'm glad to see that you're sorted. 
Please let us know if you encounter any other weirdness.
Thanks!

**Comment by weurth** [comment]
I also had this syncing issue. Win 10 64bit - 1703 update (No updates during the time I've been testing so don't think that's what caused it for me)

I set up two wallets when Breeze was first released, synced within about 10 mins and transacted between the two with no issue.

Worked fine for a day or two then got stuck very slowly syncing with 8 connections (~75%). It took about 2 days of on and off usage to sync fully. Fired it up today and it is showing 100% sync again and transactions are going through

When it was syncing it didn't matter which wallet I used, both showed the same sync percentage.

I also set up a third wallet when it wasn't syncing and that made no difference. (All 3 are working now)

Also as a side note, it took me a little while to work out it wasn't syncing because I was trying to send transactions and they weren't going but there was no message to say not synced / disconnected or something so I was a little confused initially (Same behaviour as when you put the wrong password when trying to send)
